spec/std/atomic_spec.cr

```diff
@@ -65,11 +65,11 @@ describe Atomic do
     it "with pointer" do
       atomic = Atomic.new(Pointer(Void).null)
 
-      atomic.compare_and_set(Pointer(Void).new(1), Pointer(Void).new(3)).should eq({Pointer(Void).null, false})
+      atomic.compare_and_set(Pointer(Void).new(1_u64), Pointer(Void).new(3_u64)).should eq({Pointer(Void).null, false})
```

Comment:
**thought:** All these calls with a number literal target `Pointer.new(Int)` right now, but that's only due to overload precedence. They are valid for `Pointer.new(UInt64)` as well and would resolve to that when the `Int` overload is removed.
Explicitly typed number literals are only a temporary measure to target the overload that continues to exist.
IMO that's not really necessary.
I'm fine with the change, but think we could also just leave it as it is.